src/main/java/it/gov/pagopa/payment/utils/AuditUtilities.java

```diff
@@ -0,0 +1,97 @@
+package it.gov.pagopa.payment.utils;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.util.List;
+
+@Component
+@AllArgsConstructor
+@Slf4j(topic = "AUDIT")
+public class AuditUtilities {
+    public static final String SRCIP;
+
+    static {
+        String srcIp;
+        try {
+            srcIp = InetAddress.getLocalHost().getHostAddress();
+        } catch (UnknownHostException e) {
+            log.error("Cannot determine the ip of the current host", e);
+            srcIp = "UNKNOWN";
+        }
+
+        SRCIP = srcIp;
+    }
+
+    private static final String CEF = String.format("CEF:0|PagoPa|IDPAY|1.0|7|User interaction|2| event=Payment dstip=%s", SRCIP);
+    private static final String CEF_BASE_PATTERN = CEF + " msg={}";
+    private static final String CEF_PATTERN_INITIATIVE = CEF_BASE_PATTERN + " cs1Label=initiativeId cs1={}";
+    private static final String CEF_PATTERN_TRXCODE = CEF_PATTERN_INITIATIVE + " cs2Label=trxCode cs2={}";
+    private static final String CEF_PATTERN_TRXCODE_MERCHANTID = CEF_PATTERN_TRXCODE + " cs3Label=merchantId cs3={}";
+    private static final String CEF_PATTERN_USER = CEF_PATTERN_TRXCODE + " suser={}";
+    private static final String CEF_PATTERN_REWARD_REJECIONS = CEF_PATTERN_USER + " cs3Label=reward cs3={} cs4Label=rejectionReasons cs4={}";
+    private static final String CEF_PATTERN_REWARD_REJECIONS_MERCHANTID = CEF_PATTERN_USER + " cs3Label=reward cs3={} cs4Label=rejectionReasons cs4={} cs5Label=merchantId cs5={}";
+    private static final String CEF_PATTERN_INITIATIVE_MERCHANTID = CEF_PATTERN_INITIATIVE + " cs2Label=merchantId cs2={}";
+    private static final String CEF_PATTERN_TRXCODE_USERID = CEF_BASE_PATTERN + " cs1Label=trxCode cs1={} suser={}";
+    private static final String CEF_PATTERN_TRXID_MERCHANTID = CEF_BASE_PATTERN + " cs1Label=trxId cs1={} cs2Label=merchantId cs2={}";
+
+    private void logAuditString(String pattern, String... parameters) {
+        log.info(pattern, (Object[]) parameters);
+    }
+
+    public void logCreatedTransaction(String initiativeId, String trxCode, String merchantId) {
```

Comment:
Could you sort the methods in order to put close togheter logOk and logKo?
This way it will be more faster to see them (we could also use the intelliJ "region" construct in order to group them